### DISCUSSION TITLE
Update patched 1.20.80

### DISCUSSION THREAD

**Discussion by Dxzn7** [community/community#121136] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Pls update patched 1.20.80

**Comment by ebndev** [comment]
Hi @Dxzn7,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Minecraft support.

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by phyuphyuaye1092-ux** [comment]
[aw](url)
